src/test/java/sg/dex/starfish/impl/operations/TestMemoryOperations.java

```diff
@@ -81,6 +86,11 @@ public void testPrimeAsync()  {
         test.put("input", "10");
 
         Job job = memoryOperation.invokeAsync(test);
+        String provenance= memoryOperation.getMetadata().get(Constant.PROVENANCE).toString();
+
+        Map<String,Object> provMap=JSON.toMap(provenance);
+        assertTrue(provMap.get("agent").toString().contains("opf:did:"));
```

Comment:
Please add comments to describe what is being tested. @shark8me please work with Ayush to ensure these are consistent with DEP12